src/VisualStudio/Core/Def/HACK_ThemeColorFixer.cs

```diff
@@ -116,5 +122,29 @@ public void RefreshThemeColors()
 
             targetFormatMap.SetTextProperties(classificationType, targetProps);
         }
+
+        public void SubjectBuffersConnected(IWpfTextView textView, ConnectionReason reason, Collection<ITextBuffer> subjectBuffers)
+        {
+            // DevDiv https://devdiv.visualstudio.com/DevDiv/_workitems/edit/130129:
+            //
+            // We used to schedule the RefreshThemeColors task when the Roslyn Package loaded
+            // during async solution load. Even though we scheduled the task for UI thread idle,
+            // it was possible for it to run before any files had been opened.This meant
+            // that our requests for colors and editor services would cause the composition of
+            // the editor, thus leading to a UI delay during ASL.
+            // To avoid this, we now wait until a text editor has been opened to schedule the
```

Comment:
Not sure the history story in this comment makes sense...shouldn't this have been where it was all along?